stdlib/ballerina-http/src/main/java/org/ballerinalang/net/http/compiler/WebSocketServiceCompilerPlugin.java

```diff
@@ -75,10 +75,6 @@ public void process(ServiceNode serviceNode, List<AnnotationAttachmentNode> anno
                 resources.forEach(res -> WebSocketResourceValidator
                         .validate(((BLangService) serviceNode).symbol.getName().value, res, dlog, false));
             } else if (WEBSOCKET_CLIENT_SERVICE.equals(serviceType.getTypeName().getValue())) {
-                if (serviceNode.getBoundEndpoints().size() > 0) {
```

Comment:
What is this removal doing?

Reply:
I have removed the register method of the client endpoint. Hence a client service can never be bound to an endpoint. Hence no need for this check.